yapapi/strategy.py

```diff
@@ -21,10 +23,130 @@
 SCORE_REJECTED: Final[float] = -1.0
 SCORE_TRUSTED: Final[float] = 100.0
 
+PROP_DEBIT_NOTE_INTERVAL_SEC: Final[str] = "golem.com.scheme.payu.debit-note.interval-sec?"
+PROP_PAYMENT_TIMEOUT_SEC: Final[str] = "golem.com.scheme.payu.payment-timeout-sec?"
+PROP_DEBIT_NOTE_ACCEPTANCE_TIMEOUT: Final[str] = "golem.com.payment.debit-notes.accept-timeout?"
+
+MID_AGREEMENT_PAYMENTS_PROPS = [PROP_DEBIT_NOTE_INTERVAL_SEC, PROP_PAYMENT_TIMEOUT_SEC]
+
+
+@dataclass
+class PropValueRange:
+    min: Optional[float] = None
+    max: Optional[float] = None
+
+    def __contains__(self, item: float) -> bool:
+        return (self.min is None or item >= self.min) and (self.max is None or item <= self.max)
+
+    def closest_acceptable(self, item: float):
+        if self.min is not None and item < self.min:
+            return self.min
```

Comment:
This might sound like a nitpick, but I think we should care about this :)

We have no guarantee that `self.min` is in the `PropValueRange`, because we never ensure `self.min < self.max`.
I understand this **should** be the case, but one day someone will surely mix one with the other and they'll get unexpected & unpleasant results. Or maybe they will think it's the correct way to define an "empty" range (that is, a never-accepted property), this doesn't even sound that stupid :)

Reply:
it does sound like a nitpick, but sure, good point... still, best we can do is to raise ValueError in such case... `__contains__` will work correctly then and it's impossible to clamp to such a range...